src/main.js

```diff
@@ -283,6 +283,7 @@ app.whenReady().then(async () => {
 	app.on('activate', () => {
 		if (BrowserWindow.getAllWindows().length === 0) {
 			mainWindow = createMainWindow()
+			mainWindow.show()
```

Comment:
Would it make sense to wrap mainWindow into `once` listener, like in other places? Or it won't work because of `app.on` / MacOS specific handling?

```suggestion
			mainWindow.once('ready-to-show', () => mainWindow.show()))
```